pep-0999.rst

```diff
@@ -0,0 +1,175 @@
+PEP: 999
```

Comment:
PEP 12 says i have to be given one, not just pick the next one, but it seems reasonable. maybe we should fix this in PEP 12 and fix conflicts if people try to use the same number.

PEP 12 also says we can use rst already…